langchain_dappier/retrievers.py

```
@@ -0,0 +1,212 @@
+        for doc in rec_response.response.results:
+            docs.append(
+                Document(
+                    page_content=doc.summary,
+                    metadata={
+                        "title": doc.title,
+                        "author": doc.author,
+                        "source_url": doc.source_url,
+                        "image_url": doc.image_url,
+                        "pubdata": doc.pubdate,
+                    },
+                )
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)